src/meshapi/views/panoramas.py

```diff
@@ -19,25 +19,38 @@
 PANO_DIR = "data/panoramas"
 PANO_HOST = "https://node-db.netlify.app/panoramas/"
 
-celery_app.conf.beat_schedule = {
-    "update-panoramas-hourly": {
-        "task": "tasks.update_panoramas_from_github",
-        "schedule": crontab(hour="*"),
-    },
-}
-
 
 # Raised if we get total nonsense as a panorama title
 class BadPanoramaTitle(Exception):
     pass
 
 
+class GitHubError(Exception):
+    pass
+
+
 # View called to make MeshDB refresh the panoramas.
-@api_view(["GET"])
+@api_view(["POST"])
 @permission_classes([HasPanoramaUpdatePermission])
+def update_panoramas(request):
+    try:
+        panoramas_saved, warnings = sync_github_panoramas()
+        return Response(
+            {
+                "detail": f"Saved {panoramas_saved} panoramas. Got {len(warnings)} warnings.",
+                "saved": panoramas_saved,
+                "warnings": len(warnings),
+                "warn_install_nums": warnings,
+            },
+            status=status.HTTP_200_OK,
+        )
+    except (ValueError, GitHubError) as e:
+        print(e)
+        return Response({"detail": e}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
```

Comment:
Is this allowed?
```python
try:
    bad = 1/0
except Exception as e:
    result = {"a": e}
    
import json
json.dumps(result)
```
Gives me the following error:
```
TypeError: Object of type ZeroDivisionError is not JSON serializable
```

Reply:
oops